src/test/csf/timers.h

```diff
@@ -0,0 +1,78 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012-2017 Ripple Labs Inc
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+#ifndef RIPPLE_TEST_CSF_TIMERS_H_INCLUDED
+#define RIPPLE_TEST_CSF_TIMERS_H_INCLUDED
+
+#include <chrono>
+#include <ostream>
+#include <test/csf/Scheduler.h>
+#include <test/csf/SimTime.h>
+
+namespace ripple {
+namespace test {
+namespace csf {
+
+/** Gives heartbeat of simulation to signal simulation progression
+ */
+using namespace std::chrono;
```

Comment:
`using namespace` in a header outside class or function scope is generally bad form, because anyone including the header ends up exposed to the namespaces definitions.

In this case, I would just move `using namespace std::chrono` inside your `beat` function.  For your default interval setting of `60s` you would just use the verbose `std::chrono::seconds{60}`.